DirectProgramming/DPC++FPGA/Tutorials/Features/private_copies/src/private_copies.cpp

```diff
@@ -25,14 +27,14 @@ using IntScalar = std::array<int, 1>;
 
 // Forward declare the kernel name in the global scope.
 // This FPGA best practice reduces name mangling in the optimization reports.
-template <int concurrency>
+template <int NumCopies>
```

Comment:
is `NumCopies` the correct case usage or should that be `num_copies`? What do the code sample guidelines say?

Reply:
As per: https://google.github.io/styleguide/cppguide.html#Naming "non-type template parameters should follow the rules for variable names". Updating to num_copies